rust/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM lukemathwalker/cargo-chef:latest-rust-1.72-slim-bookworm as chef
+
+# See https://github.com/LukeMathWalker/cargo-chef/issues/231.
+COPY rust-toolchain.toml rust-toolchain.toml
```

Comment:
This is super important for `cargo chef` to work properly.

Reply:
Ah, I thought something may have been off here.